graphlearn_torch/python/partition/base.py

```diff
@@ -522,11 +545,14 @@ def partition(self, with_feature=True):
 
       for etype in self.edge_types:
         graph_list, edge_pb = self._partition_graph(node_pb_dict, etype)
-        save_edge_pb(self.output_dir, edge_pb, etype)
-        for pidx in range(self.num_parts):
-          save_graph_partition(self.output_dir, pidx, graph_list[pidx], etype)
-        if with_feature:
-          self._partition_and_save_edge_feat(graph_list, etype)
+        if graph_caching:
+          save_graph_cache(self.output_dir, graph_list, etype)
```

Comment:
save edge_pd and edge_feature

Reply:
The purpose I didn't save eids and edge_pb file is that if we use graph caching, the total memory consumption is very large, I want to avoid loading eids and edge pb in igbh-full training to save memory. Considering the scenario of using edge features, can we save edge_pb and eids when partitioning and do not load them when using graph_caching and not use edge_features?